### DISCUSSION TITLE
[Project Beta] Draft Issue Improvements

### DISCUSSION THREAD

**Discussion by thefivetoes** [community/community#12989] [Projects and Issues]
I can currently create a "Draft" issue on a project board, but it feels like it is pushing me to turn this into an issue in a repo somewhere. **I'd like to use this as a task that isn't associated with a specific project**, which I CAN do currently, but here are some thoughts on how to make that more helpful:
- When I select the item on the board, I'd like to be able to get a link to the Draft Issue. That way I can create it, and share it in Slack.
- When I create a Draft Issue, I'd like to be able to assign it to someone. Currently this option is disabled. 

Seems like a possible workaround is to create a repository to capture all of these unrelated issues, but that feels messy and gross.

**Comment by BirgitPohl** [comment]
Voting definitely for links of drafts. It is difficult to talk about them, when they don't come with their own link.

Suggestion:

The boards have views and can attache also issue

`https://github.com/orgs/[org]/projects/[number]/views/[number]/issues/[number or string]`

**Comment by thibaudcolas** [comment]
A small improvement would also be for draft issue descriptions to render the same as normal issue description. For example, in a draft issue, the following Markdown doesn’t have links created to the two issues: `Depends on #8248, #4445`.

In our case, we use draft issues both as drafts, and for small tasks we don’t really want to spam people with "new issue" notifications for.

**Reply by DominusExult** [reply]
I would also apprecite it a lot  to be able to link to drafts, preferably as mentioned above with named links.
This would allow us to migrate our Trello board.

And really be able to rename it from "drafts" to "tasks" or whatever one fancies.

**Comment by vvaezian** [comment]
It seems markdown doesn't work in drafts. I had some notes in a classic project which contained links. After migration they are now draft issues, and drafts are not rendering links properly, i.e. it shows as `[link text](https://example.com)`, instead of [link text](https://example.com).
